Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -247,7 +256,7 @@ struct PerWebProcessState {
 
     BOOL _findInteractionEnabled;
 #if HAVE(UIFINDINTERACTION)
-    RetainPtr<UIView> _findOverlay;
+    Vector<RetainPtr<UIView>> _findOverlaysForOverflowScroll; // Top, right, bottom, left
```

Comment:
"OverflowScroll" is bit of an overloaded term here, since this is not really about `overflow: scroll`.

`_findOverlaysOutsideContentView` might be more accurate.